app/controllers/profile_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ProfileController < ApplicationController
+  respond_to :html
+
+  def edit
+  end
+
+  def update
+    if current_user.update(user_params)
+      redirect_to edit_profile_path,
+                  notice: 'Профиль пользователя успешно обновлен.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.